### ISSUE TITLE
Optional rounding of 

### ISSUE THREAD

**Issue by maranomynet** [stylus/stylus#2024]
When using Stylus to calculate lots of em, remand % values one ends up with tons of ridiculously precise numbers (15 digit decimal fractions) which bloats the unminified CSS.

It would be nice if Stylus would allow me to control the outgoing/final precision of these values - via an API/CLI flag.

**Comment by Scriptura** [comment]
I agree. Especially as Internet Explorer does not drop below two decimal places.
There should be a global option as it offers on the [Gulp Ruby Sass](https://github.com/sindresorhus/gulp-ruby-sass#sass-options): `precision: 2`.

Currently, for Stylus, it must specify the number of decimal places for each items. Example: `floor (value, 2)`.

I agree also that a global option would be relevant.

**Comment by kizu** [comment]
While that won't be that hard to implement, I'm not sure if this falls into Stylus' scope. That is already possible in Stylus:

1. You can manually call `round`, `floor` or `ceil` with a second argument where needed.
2. You can write all the transparent mixins for the properties where the sizes could go and then call the appropriate rounding bif there for each argument and subargument.

Also, it would be far easier to implement this using a post-processor like PostCSS.

Having those ways to round stuff, do we really need a separate global setting in Stylus for this? Won't it be a feature bloat?

**Comment by Scriptura** [comment]
@kizu: Ok, I think it's a shame but I understand your point of view. A contributor on StackOverflow I just proposed a solution through PostCSS](http://stackoverflow.com/questions/33717254/stylus-under-gulp-round-up-all-decimal-globally).
